### PR TITLE
Fix code snippets in VDSM hooks page

### PR DESCRIPTION
**Pull request by rollandf**
I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @rollandf

This pull request needs review by: @mureinik @nirs

### CODE REVIEW

File: source/develop/developer-guide/vdsm/hooks.html.md [hunk 1]
```diff
@@ -207,33 +180,45 @@ This xml can be changed in order to alter the VM settings in ways ovirt-engine d
...
-         sys.stderr.write('cantmigrate: before_vm_migrate_source: cannot migrate this VM\n')
-         sys.exit(2)
+```python
+  #!/usr/bin/python
+  import os
+  import sys 
+  if os.environ.has_key('cantmigrate'):
+     sys.stderr.write('cantmigrate: before_vm_migrate_source: cannot migrate this VM\n')
+     sys.exit(2)
+```
```
Comment (mureinik) [inline source/develop/developer-guide/vdsm/hooks.html.md]: All these lines start with two redundant spaces - please remove them.

File: source/develop/developer-guide/vdsm/hooks.html.md [hunk 2]
```diff
@@ -207,33 +180,45 @@ This xml can be changed in order to alter the VM settings in ways ovirt-engine d
...
+          if not vcpu.hasAttribute('cpuset'):
+             sys.stderr.write('pincpu: pinning cpu to: %s\n' % os.environ['pincpu'])  #sys.stderr.write is caught by vdsm and logged into vdsm.log for debugging
+             vcpu.setAttribute('cpuset', os.environ['pincpu'])  #change an attribute here
+             hooking.write_domxml(domxml)                       #and write to the altered domxml
+          else:
+             sys.stderr.write('pincpu: cpuset attribute is present in vcpu, doing nothing\n')
+      except:
+          sys.stderr.write('pincpu: [unexpected error]: %s\n' % traceback.format_exc())
+          sys.exit(2)
+```
```
Comment (mureinik) [inline source/develop/developer-guide/vdsm/hooks.html.md]: All these lines start with two redundant spaces - please remove them.

File: source/develop/developer-guide/vdsm/hooks.html.md [hunk 3]
```diff
@@ -207,33 +180,45 @@ This xml can be changed in order to alter the VM settings in ways ovirt-engine d
...
-             sys.exit(2)
+```python
+  #!/usr/bin/python 
+  import os
+  import sys
+  import hooking
+  import traceback
+      
+  pincpu usages
+  =============
```
Comment (mureinik) [inline source/develop/developer-guide/vdsm/hooks.html.md]: This should be an executable python script people can copy-paste from the site. Please comment out the "pincpu usages" 'header'

File: source/develop/developer-guide/vdsm/hooks.html.md [hunk 4]
```diff
@@ -207,33 +180,45 @@ This xml can be changed in order to alter the VM settings in ways ovirt-engine d
...
+  pincpu="0" (use the first cpu)
+  pincpu="1-4" (use cpus 1-4)
+  pincpu="^3" (dont use cpu 3)
+  pincpu="1-4,^3,6" (or all together)
+
```
Comment (mureinik) [inline source/develop/developer-guide/vdsm/hooks.html.md]: Here too - comment out the explanations

File: source/develop/developer-guide/vdsm/hooks.html.md [hunk 5]
```diff
@@ -207,33 +180,45 @@ This xml can be changed in order to alter the VM settings in ways ovirt-engine d
...
+  import os
+  import sys
+  import hooking
+  import traceback
+      
+  pincpu usages
+  =============
+  pincpu="0" (use the first cpu)
+  pincpu="1-4" (use cpus 1-4)
+  pincpu="^3" (dont use cpu 3)
```
Comment (mureinik) [inline source/develop/developer-guide/vdsm/hooks.html.md]: While you're at it: dont->don't.

**Comment by rollandf** [comment]
Fixed comments from first version.

**Comment by nirs** [comment]
:+1:

**Pull request merged**